corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -305,6 +302,7 @@ def test_error_writing_to_blobdb(self):
         error = ConnectionClosedError(endpoint_url='url')
         with self.assertRaises(ConnectionClosedError), patch.object(get_blob_db(), 'put', side_effect=error):
             self._submit('form_with_case.xml')
+        self.client.exc_info = None  # clear error to prevent it from being raised on next request
```

Comment:
why did this become necessary now?

Reply:
I don't know how it was passing on Travis. The test fails when I run it locally on a revision from before this PR. It's clear from looking at the Django test client implementation that the exception must be cleared to prevent the next request from re-raising it.